formats/mpq/mpq.go

```diff
@@ -1,22 +1,29 @@
 package mpq
 
-// #cgo LDFLAGS: -L./stormlib/ -lstorm -lz -lbz2 -lstdc++
-// #include <stdlib.h>
+// #ifdef _MPQ_WINDOWS
+// 		#include <windows.h>
+// 		#include <stdlib.h>
+// #endif
+// #ifdef _MPQ_LINUX
+//		#include <stdlib.h>
+//		#define WINAPI
```

Comment:
Should probably move all the `#define`s (except `bool`) here. Windows should already have them all defined, so they are only necessary on Linux.

Reply:
Gave it a shot, compiler complained about missing every one of them, so it seems both platforms need the defines?